demo/src/features/debug3d/extract.rs

```diff
@@ -1,28 +1,25 @@
-use crate::features::debug3d::plugin::Debug3DStaticResources;
-use crate::features::debug3d::prepare::Debug3dPrepareJobImpl;
-use crate::features::debug3d::{Debug3dRenderFeature, DebugDraw3DResource, ExtractedDebug3dData};
+rafx::declare_render_feature_extract_job!();
+
+use super::{DebugDraw3DResource, StaticResources};
 use rafx::assets::AssetManagerRenderResource;
-use rafx::nodes::{
-    ExtractJob, FramePacket, PrepareJob, RenderFeature, RenderFeatureIndex,
-    RenderJobExtractContext, RenderView,
-};
 
-pub struct Debug3dExtractJob {}
```

Comment:
I think we can remove the "Impl" suffix. It's left over from when there was another layer to the jobs that had the destiny-style callbacks.

I think we should keep the prefix (Debug3d in this case) so that we don't have N structs with the same name.

Reply:
I felt that the name prefix was unnecessary because 1.) from a code organization POV, it's already namespaced by the feature folders, and 2.) it's not accessible outside of the feature folder. No external code can try to `use` or `new` up one of these structs.

Connected to the above point, if there's no prefix, there needs to be some type of suffix so it doesn't conflict with the trait name. If we _do_ put the prefix back, then I agree the suffix can be removed.

Reply:
Per discussion offline, will add the prefix & remove the suffix.